core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -20,7 +20,7 @@
 import cloneDeep from 'lodash/cloneDeep';
 
 import { downgradeInjectable } from '@angular/upgrade/static';
-import { Injectable } from '@angular/core';
+import { Injectable, EventEmitter } from '@angular/core';
```

Comment:
When you have multiple imports from the same library, sort them.
```suggestion
import { EventEmitter, Injectable } from '@angular/core';
```

Reply:
Done